cardano-testnet/src/Testnet/Start/Types.hs

```diff
@@ -54,8 +51,8 @@ cardanoDefaultTestnetOptions = CardanoTestnetOptions
   , cardanoEpochLength = 500
   , cardanoSlotLength = 0.1
   , cardanoTestnetMagic = 42
-  , cardanoActiveSlotsCoeff = 0.1
-  , cardanoMaxSupply = 10020000000
+  , cardanoActiveSlotsCoeff = 0.05
```

Comment:
This actually was never set to 0.1 in testnet genesis and this parameter was always ignored. I've set it to the value which was always used i.e. the value from default shelley genesis in API 0.05.